themes/nav-community-v2/static/css/homepage.scss

```diff
@@ -325,54 +329,56 @@ $nav-purple-light: #9A54C4;
       display: flex;
       justify-content: center;
     }
-    
+
     .grid-container {
-      width:1200px;
+      width: 1200px;
       display: grid;
-      grid-template-columns: repeat(3,400px);
+      grid-template-columns: repeat(3, 400px);
+      // grid-auto-rows: 400px;
       justify-content: center;
-      align-items:center;
+      align-items: center;
       margin-bottom: 70px;
     }
-    
-    .grid-container>div {
+
+    .grid-container > div {
       padding: 40px;
       border: solid 1px $grey;
-      height: 350px;
       background-color: #fff;
+      height: 100%;
       text-align: center;
       img {
         height: 100px;
-        margin: 0 0 15px 0; 
+        margin: 0 0 15px 0;
       }
       h2 {
         margin: 0 0 0px 0;
+        line-height: 32px;
```

Comment:
This is the other meaningful change, the line height for this was set to 55px before (via the css cascade) which was ridiculous.